qiskit_jku_provider/qasm_simulator_jku.py

```diff
@@ -20,12 +20,11 @@
 
 from .jkujob import JKUJob
 from .jkusimulatorerror import JKUSimulatorError
-from qiskit.qobj import qobj_to_dict
 from qiskit.result import Result
 from qiskit.providers import BaseBackend
 from qiskit.providers.models import BackendConfiguration
 
-VERSION = '1.1.0'
+VERSION = '1.1.2'
```

Comment:
How do we decide the version? Do we want to start with 1.x or 0.x?

Reply:
Version 1.0 was the first working version I felt that can be exposed to IBMers. Since this is not an "official" decision we can switch to 1.0 but it might require removal of existing packages from PyPi.